lib/tables.c

```diff
@@ -22,6 +22,7 @@
 #include <assert.h>
 #include <stdbool.h>
 #include <stdlib.h>
+#include <math.h>
```

Comment:
What's math.h for? I don't think we're linking against libm at the moment, it'd be nice to keep it that way unless we need it.

Reply:
`-INFINITY` - the only way i could think to cleanly start at `j=0`